core/src/main/scala/sjsonnew/Unbuilder.scala

```diff
@@ -148,22 +148,33 @@ class Unbuilder[J](facade: Facade[J]) {
       case x => stateError(x)
     }
 
-  def nextField(): (String, J) =
+  def nextFieldOpt(): (String, Option[J]) =
     state match {
       case InObject =>
         contexts.head match {
-          case ctx: UnbuilderContext.ObjectContext[J] =>
-            // Encode elided field as JNull.
-            ctx.next match {
-              case (k, Some(v)) => (k, v)
-              case (k, None)    => (k, facade.jnull())
-            }
+          case ctx: UnbuilderContext.ObjectContext[J] => ctx.next
```

Comment:
@dwijnand Not sure if this is what you were talking about, but no more `null` or `JNull` with this solution.